### DISCUSSION TITLE
Downgrade to older Octoprint version?

### DISCUSSION THREAD

**Discussion by mc71** [feelfreelinux/octo4a#58] [Q&A]
I don't know my way around the Android file system.  Is it possible to use an older specific version of Octoprint as the install target or downgrade after install on Octo4a? I've got a plugin that has a breaking change on the latest versions of Ocotprint so for now I need to use an older version.

Something similar to this.  https://community.octoprint.org/t/how-can-i-revert-to-an-older-version-of-the-octoprint-installation-on-my-octopi-image/205

**Comment by Portaill** [comment]
did you manage to do it ?
